dir.rb

```diff
@@ -248,9 +255,30 @@ def self.[](*args, base: nil, sort: true)
   #   lib
   #   main.rb
   #
+  # If optional keyword argument +flags+ is given,
+  # the value modifies the matching; see below.
+  #
+  # If optional keyword argument +base+ is given,
+  # its value specifies the base directory.
+  # Each pattern string specifies entries relative to the base directory;
+  # the default is <tt>'.'</tt>.
+  # The base directory is not prepended to the entry names in the result:
+  #
+  #   Dir.glob(pattern, base: 'lib').take(5)
+  #   # => ["abbrev.gemspec", "abbrev.rb", "base64.gemspec", "base64.rb", "benchmark.gemspec"]
+  #   Dir.glob(pattern, base: 'lib/irb').take(5)
+  #   # => ["cmd", "color.rb", "color_printer.rb", "completion.rb", "context.rb"]
+  #
+  # If optional keyword +sort+ is given, its value specifies whether
+  # the array is to be sorted; the default is +true+.
+  # Passing value +false+ with that keyword disables sorting
+  # (though the underlying file system may already have sorted the array).
+  #
+  # <b>Patterns</b>
+  #
   # Each pattern string is expanded
   # according to certain metacharacters;
-  # examples below use the
+  # examples below use the {Ruby file tree}[rdoc-ref:Dir@About+the+Examples]):
   # {Ruby project file tree}[https://github.com/ruby/ruby]:
```

Comment:
There's sort of issue here, the `Ruby file tree` part is duplicated twice.

Reply:
Thanks for noticing; now fixed.